extensions/wikia/Wall/WallMessage.class.php

```diff
@@ -1163,18 +1166,36 @@ public function hideRelatedNotifications() {
 		}
 	}
 
+	/**
+	 * Is given Forum thread marked as closed?
+	 *
+	 * @return bool
+	 */
 	public function isArchive() {
-		return $this->isMarkInProps( WPP_WALL_ARCHIVE );
+		return $this->getCommentsIndexEntry()->isArchived();
 	}
 
+	/**
+	 * Was given Wall / Forum entry deleted via Wall's drop-down menu "remove" item?
+	 *
+	 * @return bool
+	 */
 	public function isRemove() {
-		return $this->isMarkInProps( WPP_WALL_REMOVE );
+		return $this->getCommentsIndexEntry()->isRemoved();
 	}
 
+	/**
+	 * Was given Wall / Forum entry was deleted via "VSTF Delete" button or "delete" dropdown item?
+	 *
+	 * @return bool
+	 */
 	public function isAdminDelete() {
-		return $this->isMarkInProps( WPP_WALL_ADMINDELETE );
+		return $this->getCommentsIndexEntry()->isDeleted();;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1116)